py/asmrv32.h

```diff
@@ -756,6 +757,7 @@ void asm_rv32_emit_store_reg_reg_offset(asm_rv32_t *state, mp_uint_t source, mp_
 #define ASM_STORE_REG_REG(state, rs1, rs2) ASM_STORE32_REG_REG(state, rs1, rs2)
 #define ASM_SUB_REG_REG(state, rd, rs) asm_rv32_opcode_sub(state, rd, rd, rs)
 #define ASM_XOR_REG_REG(state, rd, rs) asm_rv32_emit_optimised_xor(state, rd, rs)
+#define ASM_ZERO_REGISTER(state, reg)
```

Comment:
To match the naming of the other macros here, I suggest `ASM_CLR_REG(state, rd)`; "clear" is more of an action than "zero" which really means "zero out".

Reply:
No worries, I've updated the commit in question to use your suggestion :)